apps/web/src/components/score/ScoreConfig.ts

```diff
@@ -0,0 +1,57 @@
+export const mobileData = {
+  profile: {
+    padding: `15px 0`,
+    justifyContent: 'center',
+    containerJustifyContent: 'flex-start',
+    maxWidth: '20ch',
+    fontSize: '90%',
+    fontWeight: '',
+    textAlign: 'center',
+    textMarginLeft: '',
+    margin: '40px 0 0 0',
+  },
+  info: {
+    columns: 'repeat(2, 1fr)',
+    justifyContent: 'center',
+    margin: '15px 0 0 0',
+    gap: '12px 0',
+    width: '210px',
+  },
+  wrapper: {
+    width: '100%',
+    bottom: '7vh',
+    containerHeight: 'inherit',
+  },
+  number: {
+    transform: ``,
+    transformOrigin: '',
```

Comment:
is empty

Reply:
For the empty properties, I thought it would be useful to maintain them incase I needed to change something.
A sort of "it is here just to show this exists and can be used if needed"